app/views/shared/_modal_form.html.erb

```diff
@@ -9,12 +9,13 @@
         </div>
         <%= simple_form_for [@product, @review] do |f| %>
           <div class="modal-body">
-            <%= f.input :rating, input_html: { min: '1', max: '5' }, required: true %>
+            <%= f.input :rating, input_html: { min: '1', max: '5' }, required: true, label: 'Rating (1-5)' %>
             <%= f.input :comment, as: :text, required: true %>
+
```

Comment:
Add hidden field here. Set source to 'BarcSnap'. Do not insert empty source value into the database if it's review from our website.